code/game/objects/items/holotool.dm

```diff
@@ -0,0 +1,114 @@
+/obj/item/holotool
+	name = "experimental holotool"
+	desc = "A highly experimental holographic tool projector."
+	icon = 'icons/obj/objects.dmi'
+	inhand_icon_state = "holotool"
+	icon_state = "holotool"
+	damtype = BURN
+	force = 0
+	slot_flags = ITEM_SLOT_BELT
+	hitsound = 'sound/weapons/blade1.ogg'
+	usesound = 'sound/items/pshoom.ogg'
+	lefthand_file = 'icons/mob/inhands/items_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/items_righthand.dmi'
+	actions_types = list(/datum/action/item_action/change_tool, /datum/action/item_action/change_ht_color)
+	w_class = WEIGHT_CLASS_TINY
+
+	var/obj/item/current_tool = 0
+	var/current_color = "#48D1CC" //mediumturquoise
+
+/obj/item/holotool/proc/AddTool(typee, namee)
+	var/obj/item/WR = new typee(src)
+	WR.forceMove(src)
+	WR.name = namee
+	WR.usesound = usesound //use the same sound as we do
+	WR.toolspeed = 0.55
+
+/obj/item/holotool/proc/AddTools()
+	AddTool(/obj/item/wrench, "holo-wrench")
+	AddTool(/obj/item/screwdriver, "holo-screwdriver")
+	AddTool(/obj/item/wirecutters, "holo-wirecutters")
+	AddTool(/obj/item/weldingtool/largetank, "holo-welder")
+	AddTool(/obj/item/crowbar, "holo-crowbar")
+	AddTool(/obj/item/multitool, "holo-multitool")
```

Comment:
I'd rather have this use the tool behaviours (TOOL_WRENCH, TOOL_SCREWDRIVER, etc).
So instead of adding all these objects to the contents of the item, you make a list with all the possible tool behaviours and pick them from that.